docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-creating-a-case-proc.adoc

```diff
@@ -5,28 +5,18 @@ A case project is a type of project in {CENTRAL} that is configured for case man
 
 * Runtime strategy is set to `Per Case` in order to correctly initialize the {SERVER}. 
 * Case and documents marshallers are configured. These marshallers serialize case file data across the entire case.
-* `WorkDefinition.wid` files are created in the project and its packages when a new case definition is created to ensure case related nodes such as `Milestone` are available in the case definition design palette.
+* `WorkDefinition.wid` files are created in the project and its packages to ensure case related nodes such as `Milestone` and `Cases` are available in the case definition design palette.
+
 
-.Prerequisite
-{PRODUCT} is installed and {CENTRAL} has been deployed in a web browser.
 
 .Procedure
 
 . In the *myteam* space in {CENTRAL}, click the down arrow next to the *Add Project* button and select *Case project*.
```

Comment:
It's `mySpace` now instead of `myTeam`, per the CR build. Just fyi. Also, I'd just say ", click Add Project -> Case project" to streamline.